reflex/.templates/apps/blank/code/blank.py

```diff
@@ -16,19 +16,33 @@ def index() -> rx.Component:
     return rx.container(
         rx.color_mode.button(position="top-right"),
         rx.vstack(
-            rx.heading("Welcome to Reflex!", size="9"),
+            rx.heading("Welcome to Reflex 🦕", size="9"),
+            rx.text("Need help? 🤔 Check out the documentation."),
```

Comment:
i think this "Need help?" should appear after the "You can edit this page..." bit.